### DISCUSSION TITLE
Ereignisanzeige --> Fehlermeldung

### DISCUSSION THREAD

**Discussion by danny1605** [tbnobody/OpenDTU#1521] [Q&A]
Guten Tag, ich betreibe einen HMS1800. Am Eingang 1 und 2 hängt ein Zendure Solarflow. Sobald der HMS morgens Startet bekomme ich die Meldung PV-1-2: Kein Eingang, PV-1-2: Eingangsunterspannung. Das ist soweit ja auch richtig, da der SolarFlow HUB mit den Modulen die batterie läd. 
Jetzt zu meiner Frage besteht die Möglichkeit diese Fehlermeldung zu unterdrücken das mir diese nicht angezeigt wird ?

![Unbenannt](https://github.com/tbnobody/OpenDTU/assets/59099140/94460677-d553-4d80-b839-a4f0845a0fd0)

**Comment by DPO99** [comment]
Ich würde sagen nein, da diese Meldungen direkt vom WR ausgelesen werden. Darüber hinaus wäre es etwas sinnfrei ein Log zu schreiben und gewisse Dinge hart auszublenden. Was stört dich denn daran? Einfach ignorieren und gut ist.
